Xamarin.Forms.Platform.Android/CollectionView/ObservableItemsSource.cs

```diff
@@ -18,6 +18,9 @@ public ObservableItemsSource(IEnumerable itemSource, ICollectionChangedNotifier
 			((INotifyCollectionChanged)itemSource).CollectionChanged += CollectionChanged;
 		}
 
+
+		public event EventHandler<NotifyCollectionChangedEventArgs> CollectionItemsSourceChanged;
```

Comment:
Should this just be *NotifyCollectionChangedEventHandler* ? Should this class just be changed to implement *INotifyCollectionChanged*